apps/frontend-pwa/.env.assessment

```
@@ -3,8 +3,7 @@ API_DOMAIN=assessment-api$COOKIE_DOMAIN
 
 NEXT_PUBLIC_AVATAR_BASE_PATH="https://sos-ch-dk-2.exo.io/klicker-prod/avatars"
 NEXT_PUBLIC_IMAGE_BASE_PATH="https://sos-ch-dk-2.exo.io/klicker-prod/img"
-NEXT_PUBLIC_ADD_RESPONSE_URL="https://backend-responses$COOKIE_DOMAIN/api/AddResponse"
-NEXT_PUBLIC_ADD_RESPONSE_V2_URL="https://response-api$COOKIE_DOMAIN/AddResponse"
+NEXT_PUBLIC_ADD_RESPONSE_URL="https://assessment-response-api$COOKIE_DOMAIN/AddResponse"
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ADD_RESPONSE_URL key should go before the NEXT_PUBLIC_AVATAR_BASE_PATH key

(UnorderedKey)